### DISCUSSION TITLE
Dldi support?

### DISCUSSION THREAD

**Discussion by strikersix23** [rafaelvcaetano/melonDS-android#825] [Q&A]
Is dldi working or is it in the works

**Comment by rafaelvcaetano** [comment]
It is planned, but I'm still figuring out the best way to make it easy to use.

**Reply by teroli21-pa** [reply]
any update on it? it's been 4 years now

**Comment by strikersix23** [comment]
NooDS emulator has it implemented maybe you can take a look at it

https://github.com/Hydr8gon/NooDS

specifically

https://github.com/Hydr8gon/NooDS/blob/master/src/dldi.cpp
https://github.com/Hydr8gon/NooDS/blob/master/src/dldi.h

project is gnu gpl 3

**Comment by Haxorzz** [comment]
It has been implemented in melonDS 0.94, now this frontend needs to update their melonDS version
